voice-client-javascript/assets/admin/statuses.private.js

```diff
@@ -268,13 +351,13 @@ async function getDefaultPasswordChanged(context) {
     title: "Default admin password has been changed",
     valid: false,
   };
-  if (process.env.ADMIN_PASSWORD === "default") {
+  if (context.ADMIN_PASSWORD === "default") {
     status.description = stripIndents`
     Please take a moment to change your admin password from the provided default password. 
     
-    You can do this by editing the \`ADMIN_PASSWORD\` value in the \`.env\` in the root of this project.
+    You can do this by editing the \`ADMIN_PASSWORD\` value in your environment.
```

Comment:
I'm sorry i should have been more clear before. We can't assume a quick deploy user is going to know how to update this and redeploy. Also this is not going to mean anything to the user in the quick deploy context. 
```
\`\`\`bash
twilio serverless:deploy
\`\`\`
```


Is there any way we can help streamline/simplify this process?